app/auth/signin/page.js

```diff
@@ -110,9 +110,9 @@ const SignInPage = () => {
             </a>
           </div>
           <p className={styles.signUp}>
-            If you don't have an account,plese
-            <Link href="/register" className={styles.signUpLink}>
-              sign up
+            Don’t have an account?{" "}
```

Comment:
Delete {“ ”}. If the space is needed, use &nbsp; or {"\u00a0"}

Reply:
I have replaced it.